userspace/libsinsp/ifinfo.h

```diff
@@ -41,6 +41,8 @@ class SINSP_PUBLIC sinsp_ipv4_ifinfo
 	sinsp_ipv4_ifinfo(uint32_t addr, uint32_t netmask, uint32_t bcast, const char* name);
 
 	std::string to_string() const;
+	std::string addr_to_string(const uint32_t addr) const;
```

Comment:
The existing `to_string` method creates a string not suitable for the metrics feature. While here thought it would be good to expose an overloaded method in case you wanted to just convert for example the netmask to a string.